src/ifc-printer/main.cxx

```diff
@@ -143,5 +145,8 @@ int main(int argc, char** argv)
     catch (...)
     {
         translate_exception();
+        return_code = EXIT_FAILURE;
```

Comment:
I would suggest to just `return EXIT_FAILURE` here instead of enforcing an SESE style.

Reply:
Done!